.azure-pipelines.yml

```diff
@@ -242,3 +242,24 @@ jobs:
           sudo -E rm -rf /root/optee_repo_qemu_v8/out-br/build/optee_test*
           sudo -E make -C /root/optee_repo_qemu_v8/build arm-tf-clean
           sudo -E make -C /root/optee_repo_qemu_v8/build -j$(getconf _NPROCESSORS_ONLN) CFG_TEE_CORE_LOG_LEVEL=0 check XEN_BOOT=y
+
+  - job: QEMUv8_build_Rust
+    displayName: 'Run regression tests (Rust) in QEMUv8'
+    pool:
+      vmImage: ubuntu-18.04
+    container:
+      image: jforissier/optee_os_ci:qemuv8_check
+    steps:
+      - script: |
+          set -e -v
+          export HOME=/root
+          export LC_ALL=C
+          WD=$(pwd)
+          sudo -E bash -c "cd /root/optee_repo_qemu_v8/.repo/repo && git pull"
+          sudo -E bash -c "cd /root/optee_repo_qemu_v8 && repo sync -j 10"
+          sudo mv /root/optee_repo_qemu_v8/optee_os /root/optee_repo_qemu_v8/optee_os_old
+          sudo ln -s ${WD} /root/optee_repo_qemu_v8/optee_os
+
+          # FIXME: "make check-rust" should work on its own, but currently it causes "cargo: command not found"
+          sudo -E bash -c "make -C /root/optee_repo_qemu_v8/build -j$(getconf _NPROCESSORS_ONLN) CFG_TEE_CORE_LOG_LEVEL=0 OPTEE_RUST_ENABLE=y optee-rust"
+          sudo -E bash -c "source /root/.cargo/env && make -C /root/optee_repo_qemu_v8/build -j$(getconf _NPROCESSORS_ONLN) CFG_TEE_CORE_LOG_LEVEL=0 OPTEE_RUST_ENABLE=y check-rust"
```

Comment:
Hi @jforissier ,
I found that `source ~/.cargo/bin` in https://github.com/OP-TEE/build/blob/9eccdd56a756d7e669b80c94dabc70fa558c07af/common.mk#L521 doesn't work in some environments.`make OPTEE_RUST_ENABLE=y buildroot` will also cause ` cargo: command not found`.

I've opened a new PR for build/common.mk to fix it:
https://github.com/OP-TEE/build/pull/532

Reply:
Thanks. With your fix it looks like I can directly invoke the `check-rust` target, at least it worked when I tested with Docker. I will update this PR and hopefully things will go well in Azure too.